9.4.методы_строк_часть2/9.4.11.py

```diff
@@ -0,0 +1,6 @@
+word = str(input())
+counter = 0
+for i in range(len(word)):
+    if word[i] in "0123456789":
+        counter += 1
```

Comment:
Уже отмечал в другом PR эту ошибку - если у тебя есть строка и тебе нужно проитерироваться по каждому символу, тогда не нужно использовать индексирование:
```suggestion
for char in word:
    if char in "0123456789":
        counter += 1
```